layouts/_default/article.html

```diff
@@ -1,16 +1,19 @@
 <div class="article ">
-    <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl" style="height:212px; width:100%">
+    <a class="block" href="{{ .Permalink }}">
+        <img src="{{ .Params.image | absURL }}" alt="" class="rounded-t-xl" style="height:212px; width:100%"></a>
     <div class="bg-[#ECF6FF] rounded-b-xl ">
+
         <div class="px-6 pt-4">
             <h4 class="text-base font-bold text-[#1D65A6]">{{ .Params.category }}</h4>
             <h3 class="text-xl font-extrabold mt-3 "><a class="block h-24" href="{{ .Permalink }}">{{ .Title }}</a>
             </h3>
             <div class='art-s'>
-                <p class="mt-3 text-base font-extralight">{{ .Summary }}</p>
+                <p class="mt-3 text-base font-extralight">{{ .Summary | safeHTML | truncate 240 }}</p>
 
             </div>
 
         </div>
+
         <div class="flex justify-between items-center px-6 py-3 border-t mt-6">
             <div class="flex items-center">
                 <img src="{{ .Params.profile }}" alt="" class="rounded-full h-10 w-10">
```

Comment:
```suggestion
                <img src="{{ .Params.profile }}" alt="{{.alt}}" class="rounded-full h-10 w-10">
```

Reply:
The alt text is pulled in like this in the opensource file. Should it be like that here as well?